boltzrpc/boltzrpc.proto

```diff
@@ -409,6 +419,15 @@ message ListSwapsResponse {
     repeated ChainSwapInfo chain_swaps = 4;
 }
 
+message GetStatsRequest {
+    // which swaps to include in the stats
+    IncludeSwaps include = 1;
```

Comment:
How about making it optional and defaulting to all?

Reply:
The default value is ALL anyways, even if not optional, so I would leave it like this. Optional values are a bit annoying to deal with and I think its fine in this case.